pyvista/utilities/reader.py

```diff
@@ -988,6 +988,40 @@ def all_patch_arrays_status(self):
         return {name: self.patch_array_status(name) for name in self.patch_array_names}
 
 
+class POpenFOAMReader(OpenFOAMReader):
+    """POpenFOAM Reader for .foam files."""
+
+    _class_reader = staticmethod(_vtk.lazy_vtkPOpenFOAMReader)
+
+    @property
+    def case_type(self):
+        """Indicate whether decomposed mesh or reconstructed mesh should be read.
+
+        Returns
+        -------
+        int
+            If ``1``, reconstructed mesh should be read. If ``0``, decomposed mesh should be read.
```

Comment:
In my experience, pyvista tries to use human readable selections rather than integers.  Could we use "reconstructed" and "decomposed"?

Reply:
I agree. I tried something, please see new commits.